src/Raven.Studio/typescript/components/common/SelectionActions.tsx

```diff
@@ -0,0 +1,18 @@
+﻿import React, { ReactNode } from "react";
+import classNames from "classnames";
+import "./SelectionActions.scss";
+
+interface SelectionActionsProps {
+    active?: boolean;
+    children: ReactNode;
+}
+
+export function SelectionActions(props: SelectionActionsProps) {
+    const { active, children } = props;
+
+    return (
+        <div className={classNames("selection-actions", "rounded-pill", "bg-faded-primary", { active: active })}>
```

Comment:
you can use shorthand form:
{ active } instead of { active: active }

In general rule is:
{ a:a , b:b c:c } is the same as { a, b, c }